toolchain/check/eval.cpp

```diff
@@ -2174,15 +2174,70 @@ auto TryEvalTypedInst<SemIR::BindSymbolicName>(EvalContext& eval_context,
   return MakeConstantResult(eval_context.context(), bind, phase);
 }
 
+template <>
+auto TryEvalTypedInst<SemIR::SymbolicBindingType>(EvalContext& eval_context,
+                                                  SemIR::InstId inst_id,
+                                                  SemIR::Inst inst)
+    -> SemIR::ConstantId {
+  auto bind = inst.As<SemIR::SymbolicBindingType>();
+
+  Phase phase = Phase::Concrete;
+
+  // If we know which specific we're evaluating within and this is the type of
+  // an argument of that specific, its constant value refers to the type of the
+  // corresponding argument value.
```

Comment:
```suggestion
  // If we know which specific we're evaluating within and this is the type component of
  // a facet parameter of the generic, its constant value refers to the type component of the
  // corresponding argument value of the specific.
```
I interpreted "the type of an argument" here as meaning typeof(argument), rather than as meaning that the argument is a facet and this is the type component of its (type, witness) pair, and was confused.